news/2018-06-03-project-loved-week-of-june-3rd.md

```diff
@@ -0,0 +1,254 @@
+
+[![](/wiki/shared/news/2018-06-03-project-loved-week-of-june-3rd/osu/paradigm-shift.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Morimori Atsushi - Paradigm Shift](https://osu.ppy.sh/beatmapsets/432589) by [Avena](https://osu.ppy.sh/users/2198472)  
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+Paradigm Shift is one of Avena's most popular mapsets, and is a fitting map to mark his Loved debut. If you're a fan of his signature technical style seen in maps such as [Internet Bitch](https://osu.ppy.sh/beatmapsets/296198#osu/665149) and [Critical Crystal](https://osu.ppy.sh/beatmapsets/199535#osu/473210), then you will surely enjoy this map as well! There's a full spread of difficulties, including a fantastic Extra by [P o M u T a](https://osu.ppy.sh/users/610988), which is definitely worth checking out!
+
+[![](/wiki/shared/news/2018-06-03-project-loved-week-of-june-3rd/osu/sagashi-mono-bms-edit-.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[onoken - Sagashi Mono(BMS Edit)](https://osu.ppy.sh/beatmapsets/177187) by [Ayakura Mei](https://osu.ppy.sh/users/1083002)  
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+Sagashi Mono is another beautiful keysounded masterpiece by the mapper who brought you [Felys](https://osu.ppy.sh/beatmapsets/197747), one of the most notable keysounded maps ever. The map itself is interesting as well, featuring tricky rhythms and patterns that feel great to play despite their complexity. There isn't a lot to be said about this one except for the obvious: please **DO NOT** ignore beatmap hitsounds or you may be a little lost!!
+
+[![](/wiki/shared/news/2018-06-03-project-loved-week-of-june-3rd/osu/wonderman-bare-noize-remix-.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Tinie Tempah ft. Ellie Goulding - Wonderman (Bare Noize Remix)](https://osu.ppy.sh/beatmapsets/215978) by [Norb](https://osu.ppy.sh/users/1936576)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+Keeping up the English song remix trend, [Dubu](https://osu.ppy.sh/users/1936576) made his own intense "Dubu'step" map for Wonderman. Sitting at 6.96\*, this map was one of the toughest and most popular unranked maps of the time, and has remained to be an intersting look into old high SR 1/3 jump maps of the time. This map is more than just 1,2 back and forths, the angles in the kiai will send your aim hand into a tailspin.
```

Comment:
probably

Reply:
actually I'm not sure. "1/2" is probably more correct, but then it would be confusing because the sentence before it uses 1/3 in a different context

maybe "1-2" instead?

Reply:
huh, "1-2" would make more sense